mbq/client/contrib/tests/test_permissions.py

```diff
@@ -291,6 +289,23 @@ def test_registered_hooks_test_get_org_refs_for_permission(self):
             "get_org_refs_for_permission_completed", test_example_fn
         )
 
-        org_refs = self.client.get_org_refs_for_permission("person", "read:invoices")
+        self.client.get_org_refs_for_permission("person", "read:invoices")
 
-        test_example_fn.assert_any_call("person", "read:invoices", result=org_refs)
+        test_example_fn.assert_any_call(
+            "person",
+            "read:invoices",
+            result=sut.ConvenientOrgRefs(
+                set(["org", "org2"]), set([1, 2]), set([3, 4])
```

Comment:
superNit that we can not worry about: Python has literal syntax for sets!
```suggestion
                {"org", "org2"}, {1, 2}, {3, 4}
```